tools/version.sh

```diff
@@ -74,7 +74,7 @@ get_version_vars() {
 
     # Use git describe to find the version based on tags.
     #if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$("${git[@]}" describe --tags --match='v*' --abbrev=14 "${KUBE_GIT_COMMIT}^{commit}" 2>/dev/null); then
-    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.20.0+${KUBE_GIT_COMMIT}"); then
+    if [[ -n ${KUBE_GIT_VERSION-} ]] || KUBE_GIT_VERSION=$(echo "v1.21.0-alpha+${KUBE_GIT_COMMIT}"); then
```

Comment:
Is the `alpha` part still needed as of today?

Reply:
It comes from her: https://github.com/kubernetes/cloud-provider-gcp/pull/178#discussion_r599971852. And I think it would make sense to indicate the local build. What do you think?

Reply:
Let's keep the "alpha" then, just in case we have a chance to use `k/k` master